content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -48,6 +48,7 @@
 "LDC","Supports project specific porting efforts for the the LLVM-based D compiler"
 "libGD", "GD is an open source code library for the dynamic creation of images by programmers."
 "libjpeg-turbo","Supports project specific porting efforts of POWER8 ppc64/ppc64le"
+"LibreHealth","LibreHealth is collaborative community for free & open source software projects in Health IT"
```

Comment:
This is not on the POWER cluster. Please remove this and the other related page.